dist/tools/nacl_key_generator/generate-nacl-keys.c

```diff
@@ -0,0 +1,148 @@
+/*
```

Comment:
I think writing our own tools is fine, but make them as generic as possible.
For example, your code create "firmware keys", "server keys", and "keys.c".
Why don't you write one tool to create a keypair, and use that for both creating a "firmware key" and a "server key"?
Why not use an existing tool for binary -> .c conversion (e.g., xxd), or write a super small tool that does only that?